src/app/pages/main/components/main/main.component.html

```diff
@@ -0,0 +1,17 @@
+<div
+  class="flex h-[calc(100vh-4.75rem)] w-full flex-col items-center justify-center bg-primary text-white"
+>
+  <div class="mb-8">
+    <span class="mr-2 text-h1 font-semibold">Make your</span>
+    <span class="bg-warning text-h3 font-light text-primary"> choice </span>
+    <span class="text-h1 text-error">.</span>
+  </div>
+
+  <quiz-ui-button
+    route="/catalog"
+    state="enabled"
```

Comment:
your button component does not have 'state' input

Reply:
I initially tried to extend the button component but decided to leave it for the future, specifically when I actually need it.
And I forgot to delete the 'state'